src/GitWrapper.js

```diff
@@ -54,18 +54,26 @@ module.exports = class {
     // instantiate octokit
     octokit = new ExtendedOctokit({
       auth: this.#token,
```

Comment:
with a "#" this is new to me 🙈

Reply:
In ECMA2015, you can use `#` to denote private fields. These are completely inaccessible outside of the class. This doesn't work in browsers, and isn't fully included in spec, but it works in node js!